lexer/lexer_test.go

```diff
@@ -42,6 +42,7 @@ j--
 /*/*/
 /* This is a
    multiline comment */
+..
```

Comment:
Please add the following to test

...
. ..
.. .
....

… (a Unicode ellipsis)

My idea is to make sure nothing strange could pop out from a future refactoring